inference/README.md

```diff
@@ -7,23 +7,22 @@ The triton backend for shark.
 Install dSHARK
 
 ```
-git clone https://github.com/NodLabs/dSHARK.git
+git clone https://github.com/NodLabs/SHARK.git
```

Comment:
https://github.com/nod-ai/shark

Reply:
Isn't that the repo I made the PR to?  if you follow the link and then go to pull requests, this one is there